pyproject.toml

```diff
@@ -55,4 +55,4 @@ license-files = ["COPYRIGHT", "LICENSE"]
 where = ["python"]
 
 [tool.setuptools.package-data]
-"lsst.sdm_schemas" = ["schemas/*.yaml"]
+"lsst.sdm.schemas" = ["*.yaml"]
```

Comment:
Won't this leave out the ivoa and archive directories? Have you seen what pip installs?

Reply:
It needs to be `"lsst.sdm.schemas" = ["**/*.yaml"]` - I will fix.